src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,41 @@
+package duke;
+
+public class DukeException extends Exception {
+    private String command;
+    private boolean hasTime;
+
+    public DukeException(String command) {
+        this.command = command;
+        if (!command.contains("/at") || !command.contains("/by")) {
+            hasTime = false;
+        }
+    }
+
+    private String error() {
+        return "☹ OOPS!!! I'm sorry, but I don't know what that means :-(\n";
+    }
+
+    private String inCompleteMessage() {
```

Comment:
I believe this should be `incompleteMessage()` instead of `inCompleteMessage()`? Both give off very different meanings.